socorro/schemas/processed_crash.schema.yaml

```diff
@@ -2457,6 +2464,14 @@ properties:
     description: Unique identifier of this crash report.
     type: string
     permissions: ["public"]
+  crash_inconsistencies:
+    description: >
+      A list of inconsistencies detected by comparing crash reason/crash
+      address with crashing instruction and memory information.
+    type: ["array", "null"]
+    items:
+      type: string
+    permissions: ["public"]
```

Comment:
I thought this line was supposed to make the field show up in the details tab when I'm not logged in, but in dev it's only showing up when I'm logged in and I'm not sure what to change

Reply:
ah, didn't specify it enough times